### PR TITLE
🐛 FIX: Node comments API

### LINKED ISSUE #4759: node.get_comment(1) fails
### Describe the bug

When accessing node comments, I got the following error:
```python
node.get_comment(1)


---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/opt/conda/lib/python3.7/site-packages/aiida/orm/implementation/querybuilder.py in get_column(self, colname, alias)
    400         try:
--> 401             return getattr(alias, colname)
    402         except AttributeError:
/opt/conda/lib/python3.7/site-packages/sqlalchemy/orm/util.py in __getattr__(self, key)
    505             # maintain all getattr mechanics
--> 506             attr = getattr(target, key)
    507 
AttributeError: type object 'DbComment' has no attribute 'pk'
During handling of the above exception, another exception occurred:
InputValidationError                      Traceback (most recent call last)
<ipython-input-32-c82a3a4087b8> in <module>
----> 1 c=a.get_comment(1)
/opt/conda/lib/python3.7/site-packages/aiida/orm/nodes/node.py in get_comment(self, identifier)
    690         :return: the comment
    691         """
--> 692         return Comment.objects.get(dbnode_id=self.pk, pk=identifier)
    693 
    694     def get_comments(self):
/opt/conda/lib/python3.7/site-packages/aiida/orm/entities.py in get(self, **filters)
    134         """
    135         res = self.query(filters=filters)
--> 136         return res.one()[0]
    137 
    138     def find(self, filters=None, order_by=None, limit=None):
/opt/conda/lib/python3.7/site-packages/aiida/orm/querybuilder.py in one(self)
   2177         from aiida.common.exceptions import MultipleObjectsError, NotExistent
   2178         self.limit(2)
-> 2179         res = self.all()
   2180         if len(res) > 1:
   2181             raise MultipleObjectsError('More than one result was found')
/opt/conda/lib/python3.7/site-packages/aiida/orm/querybuilder.py in all(self, batch_size, flat)
   2250         :returns: a list of lists of all projected entities.
   2251         """
-> 2252         matches = list(self.iterall(batch_size=batch_size))
   2253 
   2254         if not flat:
/opt/conda/lib/python3.7/site-packages/aiida/orm/querybuilder.py in iterall(self, batch_size)
   2207         :returns: a generator of lists
   2208         """
-> 2209         query = self.get_query()
   2210 
   2211         for item in self._impl.iterall(query, batch_size, self._attrkeys_as_in_sql_result):
/opt/conda/lib/python3.7/site-packages/aiida/orm/querybuilder.py in get_query(self)
   2086 
   2087         if need_to_build:
-> 2088             query = self._build()
   2089             self._hash = queryhelp_hash
   2090         else:
/opt/conda/lib/python3.7/site-packages/aiida/orm/querybuilder.py in _build(self)
   1936                     'The tags I know are:\n{}'.format(tag, self.tag_to_alias_map.keys())
   1937                 )
-> 1938             self._query = self._query.filter(self._build_filters(alias, filter_specs))
   1939 
   1940         ######################### PROJECTIONS ##########################
/opt/conda/lib/python3.7/site-packages/aiida/orm/querybuilder.py in _build_filters(self, alias, filter_spec)
   1361                 is_attribute = (attr_key or column_name in ('attributes', 'extras'))
   1362                 try:
-> 1363                     column = self._impl.get_column(column_name, alias)
   1364                 except InputValidationError:
   1365                     if is_attribute:
/opt/conda/lib/python3.7/site-packages/aiida/orm/implementation/querybuilder.py in get_column(self, colname, alias)
    407                     colname,
    408                     alias,
--> 409                     '\n'.join(alias._sa_class_manager.mapper.c.keys())  # pylint: disable=protected-access
    410                 )
    411             )
InputValidationError: pk is not a column of aliased(DbComment)
Valid columns are:
id
uuid
ctime
mtime
content
dbnode_id
user_id
```

I appears to me that the fix of this issue is replacing:
https://github.com/aiidateam/aiida-core/blob/e794287218a5d3b85a85ba36eed6d6928a849a26/aiida/orm/nodes/node.py#L692

with:

```python
        return Comment.objects.get(dbnode_id=self.pk, id=identifier)
```

### PR DESCRIPTION
**Pull request by chrisjsewell**
closes #4759

### CODE REVIEW

**Comment by chrisjsewell** [comment]
next time just fix it 😜

File: tests/orm/node/test_node.py [hunk 1]
```diff
@@ -824,6 +824,51 @@ def test_delete_collection_outgoing_link(self):
...
 
 
+@pytest.mark.usefixtures('clear_database_before_test')
+class TestNodeComments:
+    """Tests for creating comments on nodes."""
+
+    def test_add_comment(self):
+        """Test comment addition."""
+        data = Data().store()
+        content = 'sasha is lazy'
```
Comment (yakutovicha) [inline tests/orm/node/test_node.py:834]: better write something about the dark side.

**Review by yakutovicha — CHANGES_REQUESTED** [review]
I see only one problem in this PR.

File: tests/orm/node/test_node.py (hunk 1 above)
Comment (chrisjsewell) [inline tests/orm/node/test_node.py:834]:
```suggestion
        content = 'whatever Trevor'
```

**Review by yakutovicha — APPROVED** [review]

**Pull request merged**